georgian/README.md

```diff
@@ -0,0 +1,9 @@
+# Georgian
+
+There is no official standard for Georgian braille. However a group of
+very dedicated people from the Georgian Braille printing house and the
+Georgian School For Blind maintain a de-facto standard.
+
+- Origins and history of [Georgian Braille](Georgian Braille.docx)
+- The Georgian book on [Teaching braille](ბრაილის შრიფტის სწავლება -.docx)
```

Comment:
I think we need a little bit more information about what this document is, and why we decided to include it here on out braille specifications website.

"The Georgian book on Teaching braille" sounds a bit weird IMO. The title of the document is "ბრაილის შრიფტის სწავლება _ მეთოდები და სტრატეგიები" (translated "Teaching braille - methods and strategies"), maybe it's better to use that. It seems that this is a book for braille teachers, but it's very broad. It's even more general than braille. By no means it attempts to explain the braille rules for Georgian. So the goal of including this document on our website can at most be to provide an example of an "official" text book, to show the braille rules in practice. But of course this requires that we also include the braille transcription of the book. I'm not sure if it would have much added value though, because the text only includes the letters of the Georgian alphabet and some basic punctuation.